libbeat/scripts/generate_template.py

```diff
@@ -114,6 +116,16 @@ def fill_field_properties(field, defaults):
         properties[field["name"]] = {
             "type": "date"
         }
+    elif field.get("type") == "string" and field["name"] == "message":
+        properties[field["name"]] = {
+            "type": "string",
+            "index": "analyzed"
+        }
+    elif field.get("type") == "long":
+        properties[field["name"]] = {
+            "type": "long",
+            "doc_values": "true"
```

Comment:
It is in ES 2.X, but I think it wasn't in 1.7. So I think we still want to keep it there for a while.